containers/azure-terraform/README.md

```diff
@@ -51,7 +55,43 @@ Beyond `git`, this `Dockerfile` includes `zsh`, [Oh My Zsh!](https://ohmyz.sh/),
 
 4. After following step 2 or 3, the contents of the `.devcontainer` folder in your project can be adapted to meet your needs.
 
-5. Finally, press <kbd>F1</kbd> and run **Remote-Containers: Reopen Folder in Container** to start using the definition.
+5. Open the Command Palette with `Ctrl/CMD+Shift+P` or press <kbd>F1</kbd> and run **Remote-Containers: Reopen Folder in Container** to start using the definition.
+
+## Testing the Definition
+
+This definition includes some test code that will help you verify it is working as expected on your system. Follow these steps:
+
+1. Follow steps 1-4 from the above [section](#adding-the-definition-to-your-project).
+2. Fill in the values for the environment variables in the [`.devcontainer/devcontainer.env` file](https://code.visualstudio.com/docs/remote/containers-advanced#_option-2-use-an-env-file)
+   - This file allows customization of the environment variables and the values needed for the terraform tasks.
+
+3. Open the Command Palette with `Ctrl/CMD+Shift+P` or press <kbd>F1</kbd> and run **Remote-Containers: Reopen Folder in Container** to start using the definition.
+
+4. VS Code tasks have been configured to run commonly used Terraform commands. The `test-project` folder includes a Terraform template that provisions a new Azure Resource Group and the commands can be run via `Ctrl/CMD+Shift+P` > `Tasks: Run Tasks`.
+   ![Run Terraform Tasks](test-project/assets/Terraform_tasks.png)
+
+A more detailed explanation of the tasks can be found in the next [section](#running-the-tasks).
+
+## Running the Tasks
+
+Once the environment settings are configured with the `.devcontainer/devcontainer.env`, you can begin executing terraform commands. VS Code tasks have been configured to run each of the commonly used terraform commands.
+
+- `az login`: login to Azure and set your default subscription
```

Comment:
Use Terraform Azure commands

Reply:
Clarification provided in the previous comment